src/site/stages/build/process-cms-exports/schemas/output/node-press_releases_listing.js

```diff
@@ -13,6 +16,25 @@ module.exports = {
     fieldMetaTitle: { type: 'string' },
     fieldOffice: { $ref: 'output/node-health_care_region_page' },
     fieldPressReleaseBlurb: { type: 'string' },
```

Comment:
```suggestion
    fieldPressReleaseBlurb: { oneOf: [{ type: 'string' }, { type: 'null' }] },
```

When I ran `node script/cms/test-bundle-transformers.js --bundle "node-press_releases_listing"` I received the error below. This fixes it. 
```shell
node.09e2d180-403b-482c-b8a4-1c44256060b2 (node-press_releases_listing) is invalid after transformation:
{
  "keyword": "type",
  "dataPath": ".fieldPressReleaseBlurb",
  "schemaPath": "#/properties/fieldPressReleaseBlurb/type",
  "params": {
    "type": "string"
  },
  "message": "should be string"
}
Data found at .fieldPressReleaseBlurb: null
-------------------
/Users/theodurbentum/Development/vets-website/src/site/stages/build/process-cms-exports/index.js:105
    throw new Error(`${toId(entity)} is invalid after transformation`);
    ^
```

Reply:
Good catch